### PR TITLE
Manual Tests

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

File: test/manual-tests/test-add.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        <h1>Manual Test: Add a popcorn event</h1>
+        <p> This test makes sure that popcorn events can be added without incident.</p>
+      </div>
+
+      <!-- First, the steps that the user must follow -->
+      <div id="manual-test-steps">
+        <h2>Steps to Follow:</h2>
+        <ol>
+          <li>Wait for page to load, make sure HTML5 video is displayed on page
+          <li>Drag the "footnote" plugin from Myevents (lower right) into the "area 1" div
```
Comment (mjschranz) [inline test/manual-tests/test-add.html:82]: fix spacing for myevents to make it two words.

File: test/manual-tests/test-add.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          <li>In the dialogue that appears in the tray, enter arbitrary text
+          <li>click "done" to create the event
+          <li>Drag the "image" plugin from Myevents (lower right) into the "area 2" div
```
Comment (mjschranz) [inline test/manual-tests/test-add.html:85]: Same change as above.

File: test/manual-tests/test-add.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          <li>In the dialogue that appears in the tray, paste this URL <a href="http://mozillapopcorn.org/wp-content/uploads/2011/11/icon_popcorn_js-150x150.png">http://mozillapopcorn.org/wp-content/uploads/2011/11/icon_popcorn_js-150x150.png</a> into the source URL field
+          <li>click "done" to create the event
+          <li>Scroll the playhead to a time before the events start
+          <li>PLay through
```
Comment (mjschranz) [inline test/manual-tests/test-add.html:89]: uncapitalize the L

File: test/manual-tests/test-delete.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    </div>
+    <div id="target-div"></div>
+
+    <!-- This is the test portion of the page -->
+    <div id="manual-test">
+
+      <!-- An explanation of what the test is all about -->
+      <div id="manual-test-desc">
+        <h1>Manual Test: Delete a popcorn event</h1>
+        <p> This test makes sure that popcorn events can be added without incident.</p>
```
Comment (mjschranz) [inline test/manual-tests/test-delete.html:74]: Change to deleted without incident rather than added without incident.

**Comment by jbuck** [comment]
Closed by the manual testing ticket whose number I forget.

**Pull request closed without merging**